src/System.Net.Http/src/System/Net/Http/SocketsHttpHandler/Http2Connection.cs

```diff
@@ -484,7 +484,7 @@ private void ProcessSettingsFrame(FrameHeader frameHeader)
 
                 // Send acknowledgement
                 // Don't wait for completion, which could happen asynchronously.
-                Task ignored = SendSettingsAckAsync();
+                _ = SendSettingsAckAsync();
```

Comment:
Dumb question: why is this better?

Reply:
Not a dumb question. It's the exact same resulting IL, so from a results/performance perspective it's not better.  It's just taking advantage of the C# discards feature so you don't need to name something you're not using.  Then as the discards feature gets broader adoption, it should become obvious what this is doing, if it's not already.